packages/demo/src/common/Collection/signals/init.js

```diff
@@ -0,0 +1,33 @@
+import {input, merge, set, state, when} from 'cerebral/operators'
+import {onChildAdded, onChildChanged, onChildRemoved, value} from 'cerebral-provider-firebase'
+import paths from '../paths'
+
+export default function init (moduleName, initState = {}) {
+  const {collectionPath, dynamicPaths, errorPath} = paths(moduleName)
+  return [
+    // prepare remote and local path
+    ...dynamicPaths,
+    // start listening
+    onChildAdded(
+      input`remoteCollectionPath`, `${moduleName}.updated`),
```

Comment:
@gaspard what I usually do here is pass an option to get older values, like that:

```js
{
  orderByChild: 'updated_at',
  startAt: (new Date()).getTime(),
}
```

Example: https://github.com/saitodisse/scrap-cerebral-2/blob/master/src/shared_actions/firebase/firebase_listen.js#L1


Docs: https://github.com/cerebral/cerebral/tree/master/packages/cerebral-provider-firebase#onchildadded

That way we only get new items when they are added. Bellow we are calling `value(input`remoteCollectionPath`), {` so all item will be downloaded. That's why I put the `updated_at` prop. I believe this is the cause for some error on initialization of `createOnChildAdded.js`